package.json

```diff
@@ -9,65 +9,64 @@
   },
   "author": "Boris Povod <boris@crypti.me>, Pavel Nekrasov <landgraf.paul@gmail.com>, Oliver Beddows <oliver@lisk.io>",
   "dependencies": {
-    "async": "=2.0.1",
-    "body-parser": "=1.15.2",
+    "async": "=2.1.4",
+    "body-parser": "=1.16.0",
     "bytebuffer": "=5.0.1",
-    "change-case": "=3.0.0",
+    "change-case": "=3.0.1",
     "colors": "=1.1.2",
     "commander": "=2.9.0",
     "compression": "=1.6.2",
     "cors": "=2.8.1",
     "decompress-zip": "LiskHQ/decompress-zip#f46b0a3",
-    "ejs": "=2.5.2",
-    "express": "=4.14.0",
+    "ejs": "=2.5.5",
+    "express": "=4.14.1",
     "express-domain-middleware": "=0.1.0",
     "express-query-int": "=1.0.1",
-    "express-rate-limit": "=2.5.0",
+    "express-rate-limit": "=2.6.0",
     "extend": "=3.0.0",
     "ip": "https://github.com/LiskHQ/node-ip/tarball/ec401f16915024450bf9b843eeb4c5366ef5235c",
     "json-schema": "=0.2.3",
     "json-sql": "LiskHQ/json-sql#27e1ed1",
     "lisk-sandbox": "LiskHQ/lisk-sandbox#162da38",
-    "lodash": "=4.16.2",
-    "method-override": "=2.3.6",
-    "npm": "=2.15.10",
-    "pg-monitor": "=0.5.11",
+    "lodash": "=4.17.4",
+    "method-override": "=2.3.7",
+    "npm": "=4.1.2",
```

Comment:
Let's leave `npm` as is for now. If I remember there was definite reason for using 2.*.